base/src/expressions/parser/mod.rs

```diff
@@ -202,8 +209,8 @@ impl Parser {
         let lexer = lexer::Lexer::new(
             "",
             lexer::LexerMode::A1,
-            get_locale("en").expect(""),
-            get_language("en").expect(""),
+            get_locale("en").expect("Failed to get locale"),
+            get_language("en").expect("Failed to get language"),
```

Comment:
Thanks! We actually need to remove all expect from code, but that's a problem for a different day. Here expect, expect with message or unwrap are all equally bad. Let me take care of this.